app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :rememberable, :validatable, :trackable
```

Comment:
This where the authentication options for the user are defined. I've removed the default Recoverable option as I expect we're be aiming to manually invite users and for them to log in via google. So no need for a user to recover a forgotten password.

I've also added Trackable so get a little data on user activity out of the box.

Otherwise this is the default Devise authentication set up storing users in the database and authenticating via email address and password.